api-schema/src/main/java/org/stellar/anchor/api/asset/StellarAssetInfo.java

```diff
@@ -1,14 +1,45 @@
 package org.stellar.anchor.api.asset;
 
 import com.google.gson.annotations.SerializedName;
-import lombok.Data;
-import org.stellar.anchor.api.sep.AssetInfo;
-
-@Data
-public class StellarAssetInfo extends AssetInfo {
+import org.stellar.anchor.api.sep.AssetInfo.DepositWithdrawInfo;
+import org.stellar.anchor.api.sep.AssetInfo.Schema;
+import org.stellar.anchor.api.sep.operation.Sep31Info;
+import org.stellar.anchor.api.sep.operation.Sep38Info;
+import org.stellar.anchor.api.sep.sep31.Sep31InfoResponse;
+import org.stellar.anchor.api.sep.sep38.InfoResponse;
 
+public class StellarAssetInfo implements AssetInfo {
+  String code;
   String issuer;
 
   @SerializedName("distribution_account")
   String distributionAccount;
+
+  @SerializedName("significant_decimals")
+  Integer significantDecimals;
+
+  DepositWithdrawInfo sep6;
```

Comment:
Is there a reason to use different class name from the variable name? 

ex: `Sep6Info sep6`

Reply:
Because sep6 and sep24 share the same attributes